README.adoc

```diff
@@ -407,6 +407,19 @@ Retrieve the work item.
 $ ./bin/alm-cli show workitem --id 1 -H localhost:8080 --pp
 ----
 
+=== Reset Database
+
+The database are kept in a docker container that gets reused between restarts. Thus restarts will not clear out the database.
+
+To clear out the database kill the database like this:
```

Comment:
This is completely optional and depending on how you start the DB. For instance, locally I start it with `make integration-test-env-prepare`.  And when I tear it down with `make integration-test-env-tear-down` the data is gone the next time I start the DB.

As far as I remember there's no other section in this README that tells you how to start the database. So why assume, it was started as a container?